demos/python_demos/face_recognition_demo/requirements.txt

```diff
@@ -1,3 +1,4 @@
 opencv-python>=3.4.0
 numpy>=1.11.0
 scipy>=1.1.0
+Pillow>=6.0.0
```

Comment:
Why was this added? I did not notice that the demo used something from that package.